scss/bootstrap/_modal.scss

```diff
@@ -62,19 +64,19 @@
   right: 0;
   bottom: 0;
   left: 0;
-  z-index: $zindex-modal-bg;
-  background-color: $modal-backdrop-bg;
+  z-index: 1040;
+  background-color: #000;
 
   // Fade for backdrop
   &.fade { opacity: 0; }
-  &.in { opacity: $modal-backdrop-opacity; }
+  &.in { opacity: .5; }
 }
 
 // Modal header
 // Top section of the modal w/ title and dismiss
 .modal-header {
-  padding: $modal-title-padding;
-  border-bottom: $modal-header-border-width solid $modal-header-border-color;
+  padding: 15px;
```

Comment:
Is this a standard padding size?

Reply:
Again, this has been borrowed from Moodle's core `_variables.scss` as is. It works well with this exact size so have borrowed as is.